packages/migrate/Migration.js

```diff
@@ -23,6 +23,14 @@ class Migration {
     this.config = config || {};
   }
 
+  static getNumber(fileName) {
+    const match = fileName.match(/\d+/);
```

Comment:
This regex is not equivalent to parseInt which introduces subtle changes to the migrations behavior. `Regex.match` will match the first group of digits in a string *wherever* they occur while `parseInt` will only match sequential digits at the beginning of a string.

Reply:
oh right, i'll fix it right away

Reply:
done :D